### DISCUSSION TITLE
Any way to reuse this literature lockscreen clock?

### DISCUSSION THREAD

**Discussion by dibbockb** [Mahmud0808/Iconify#533] [Q&A]
![photo_2024-07-01_11-57-21](https://github.com/Mahmud0808/Iconify/assets/47898254/9d4e967c-03fe-4b7c-8510-58ea0e0c7c0e)

This clock style was so good yet unfortunately it was removed on the newer version. is there any way to copy this from old version and use it on newer?
I humbly request to bring back atleast this lockscreen clock. it was one of a kind. 
Thanks for your precious time!

**Comment by Mahmud0808** [comment]
Added https://github.com/Mahmud0808/Iconify/commit/83eada3df0e08298a6b1473c75319b06cc305a94
